lib/absinthe/adapter/underscore.ex

```diff
@@ -1,6 +1,6 @@
 defmodule Absinthe.Adapter.Underscore do
   @moduledoc """
-  Underscores external input and leaves external input alone. Unlike the
+  Underscores internal input and leaves external input alone. Unlike the
```

Comment:
I don't think this is accurate. It underscores `external` input via `to_internal_name`.